invenio_records/tasks/index.py

```diff
@@ -30,6 +30,21 @@
 from ..signals import before_record_index
 
 
+@before_record_index.connect
+def populate_authors(recid, json):
+    """ Populates a json record before indexing it to elastic.
+        Adds a field for all the possible variations of an authors name
+    """
+    from invenio_records.tasks.tokenizer import BibIndexAuthorTokenizer
+    tokenizer = BibIndexAuthorTokenizer()
+    authors = json.get("authors")
+    if authors is not None:
+        for author in authors:
+            name = author.get("full_name")
+            if name is not None:
+                author.update({"name_variations": tokenizer.tokenize(name)})
```

Comment:
@Panos512 I notice only now. This receiver should not be implemented in invenio-records but in our overlay in `inspire.module.authors.receivers`.